test_files/basic.untyped.js

```diff
@@ -0,0 +1,17 @@
+/** @return { ?} */ // This test is just a random collection of typed code, to
+// ensure the output is all with {?} annotations.
+function func(/** ? */ arg1) {
```

Comment:
Would you believe we don't get this right even in the typed case?  I just checked!